module/zfs/mmp.c

```diff
@@ -242,9 +243,12 @@ mmp_write_done(zio_t *zio)
 	spa_t *spa = zio->io_spa;
 	vdev_t *vd = zio->io_vd;
 	mmp_thread_t *mts = zio->io_private;
+	uint64_t mmp_kstat_id;
+	hrtime_t mmp_write_duration;
 
 	mutex_enter(&mts->mmp_io_lock);
-	vd->vdev_mmp_pending = 0;
+	mmp_kstat_id = vd->vdev_mmp_kstat_id;
+	mmp_write_duration = gethrtime() - vd->vdev_mmp_pending;
```

Comment:
Since we're all in with c99 now we can safely declare these values here to be more concise.

```
uint64_t mmp_kstat_id = vd->vdev_mmp_kstat_id;
hrtime_t mmp_write_duration = gethrtime() - vd->vdev_mmp_pending;
```

Reply:
Done